guessit/rules/common/quantity.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Quantity
+"""
+import re
+
+import six
+
+
+class Quantity(object):
```

Comment:
I'm leaving the remaining PRs open to be reviewed and get feedback since there are several ways to go and. So, you should enjoy your summer break. :-p

I named it Quantity to be able to use in other future properties like bitrate (320 kbps). The Quantity name was inspired by python pint: https://pint.readthedocs.io/en/latest/

Still we can have a base class with a proper explicit name if needed.